tests/phpunit/class-wc-stripe-helper-test.php

```
@@ -1975,4 +1975,70 @@ public function provide_test_build_line_items(): array {
+				'post_type'    => 'page',
+				'post_status'  => 'publish',
+				'post_content' => $page_content,
+			]
+		);
+
+		update_option( 'woocommerce_checkout_page_id', $page_id );
+		// Isolate the assertion to the checkout page so an unrelated cart page does not interfere.
+		update_option( 'woocommerce_cart_page_id', 0 );
+
+		$this->assertSame( $expected, WC_Stripe_Helper::is_express_checkout_button_style_overridden() );
+	}
```

PHPMD (2.15.0)

[error] The method test_is_express_checkout_button_style_overridden is not named in camelCase. (undefined)

(CamelCaseMethodName)